app/components/A/tests/index.test.js

```diff
@@ -22,7 +22,7 @@ describe('<A />', () => {
     expect(renderedComponent.type()).toEqual('a');
   });
 
-  it('should have an href attribute', () => {
+  it('should have a href attribute', () => {
```

Comment:
`h` in english is treated like a vocal at the beginning of words. You say "AN apple" but you say "A banana" because apple starts with a vocal – and `h` is treated like a vocal at the beginning of words, so this should be "AN href"